src/index.ts

```diff
@@ -270,7 +272,7 @@ const dev = async ({
           },
           clientLogLevel: "none",
           injectClient: false,
-          hot: false,
+          hot: hotReloading,
```

Comment:
happy to include it, but found a ton of issues when trying to use it with Roam. mainly, it still refreshed the whole browser, which is what I was trying to avoid. Down to support it if you find it useful!